python/requests/best-practice/use-timeout.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
+- id: use-timeout
+  patterns:
+  - pattern-not: requests.Session(...)
+  - pattern-not: requests.session(...)
+  - pattern-not: requests.append(...)
+  - pattern-not: requests.$W(..., timeout=$N, ...)
```

Comment:
Does it make more sense to just enumerate the apis that need a timeout? (Is it more than the 4 HTTP methods?)

Reply:
At this point, probably 👍